infra/recipes/terraform/azure/environments/container/main.tf

```diff
@@ -0,0 +1,8 @@
+resource "azurerm_resource_group" "openhouse_sandbox" {
```

Comment:
Add a descriptive comment on purpose of the container environment in the beginning of the file

Reply:
Done